node_modules/@angular/cli

```diff
@@ -0,0 +1 @@
+../../../../../usr/local/lib/node_modules/@angular/cli
```

Comment:
same deal as `node_modules/.bin/ng`; there should not be a project-level node modules